RGPosadas/LitterRally
# Allow easy host change

closes #217 

The host.env.ts file was added to allow the developer to easily change the hostname where their application is being served, to allow for use cases where they don't want to run on localhost. For example, if they want other devices on their local network to be able to access the application, using localhost won't work, and they would need to host in on their machine's IP address for example

The README was updated to include info on how this mechanism works

The reason I went with a .ts file instead of adding these variables to our .env file is that the frontend cannot read the environment variables of our web server, since it is served to the client's browser. My solution, therefore, involved creating a ts in our library, which would allow us to set these values for both apps without hardcoding them.

How to test:
1. Follow instructions in the readme
2. Set your hostname to localhost and make sure everything works as before
3. Set your hostname to something else (for example, your machine's IP address) and test if you can access the app from other devices). Don't forget the --host flag!
